### PR TITLE
GH-904 Choropleth Tooltips displaying "undefined"

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-904

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo Please Review

File: src/map/ChoroplethCounties.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
         this.columns(["County", "Weight", "Label"]);
```
Comment (mlzummo) [inline src/map/ChoroplethCounties.js]: columns and data index count doesnt match?

Comment (mlzummo) [inline src/map/ChoroplethCounties.js]: columns and data index count doesnt match?

Comment (GordonSmith) [inline src/map/ChoroplethCounties.js]: Fixed (but it was only in testData anyway)

**Comment by mlzummo** [comment]
looks good

**Pull request merged**